src/main/java/wolox/training/models/Book.java

```diff
@@ -1,5 +1,6 @@
 package wolox.training.models;
 
```

Comment:
Complete the @Colum annotation of the genre field.

In the set method of the same field add preconditions where it is verified that the field is not null or empty.

Reply:
did you mean @Column ?

Reply:
JAJAJAJAJAJAJAJAJAJAJAJAJAJA